help/destinations/catalog/email-marketing/sendgrid.md

```diff
@@ -0,0 +1,250 @@
+
+![](../../assets/catalog/email-marketing/sendgrid/04.jpg)
+
+* Upon clicking *Connect to destination*, you will be shown the next [step](#connection-parameters-parameters) to fill in additional information fields.
+
+### Connection parameters {#parameters}
+
+While [setting up](https://experienceleague.adobe.com/docs/experience-platform/destinations/ui/connect-destination.html?lang=en) this destination, you must provide the following information:
+
+*  **[!UICONTROL Name]**: A name by which you will recognize this destination in future.
+*  **[!UICONTROL Description]**: An optional description that will help you identify this destination in future.
+
+![](../../assets/catalog/email-marketing/sendgrid/05.jpg)
+
+![](../../assets/catalog/email-marketing/sendgrid/06.jpg)
+
+## Connect to destination {#connect}
+
+* To connect to this destination, the generic steps described in the [destination configuration tutorial](https://experienceleague.adobe.com/docs/experience-platform/destinations/ui/connect-destination.html?lang=en) will provide guidance. See below for the screengrabs corresponding to this destination. 
+
+![](../../assets/catalog/email-marketing/sendgrid/07.jpg)
+
+![](../../assets/catalog/email-marketing/sendgrid/08.png)
+
+* You will be navigated to the list of destinations.
+
+![](../../assets/catalog/email-marketing/sendgrid/09.jpg)
+
+* Select the newly created destination and click *Next*.
+
+![](../../assets/catalog/email-marketing/sendgrid/10.jpg)
+
+* Select the *Segment* *(created earlier in [prerequisites](#prerequisites-prerequisites))* and click *Next*.
+
+![](../../assets/catalog/email-marketing/sendgrid/11.jpg)
+
+* Now map the *Source* and *Schema* *(created earlier in [prerequisites](#prerequisites-prerequisites))* fields.
+
+![](../../assets/catalog/email-marketing/sendgrid/12.jpg)
+
+> [!IMPORTANT]
+> The *Source* field attribute *"Email"* should be mapped to the *Target* field attribute "external_id" as shown in the subsequent steps.
+![](../../assets/catalog/email-marketing/sendgrid/13.png)
+
+![](../../assets/catalog/email-marketing/sendgrid/14.jpg)
+
+![](../../assets/catalog/email-marketing/sendgrid/15.jpg)
+
+![](../../assets/catalog/email-marketing/sendgrid/16.jpg)
+
+![](../../assets/catalog/email-marketing/sendgrid/17.jpg)
+
+![](../../assets/catalog/email-marketing/sendgrid/18.jpg)
+
+![](../../assets/catalog/email-marketing/sendgrid/19.jpg)
+
+* A complete list of the mappings to be setup is shown below.
+
+![](../../assets/catalog/email-marketing/sendgrid/20.png)
+
+![](../../assets/catalog/email-marketing/sendgrid/21.png)
+
+![](../../assets/catalog/email-marketing/sendgrid/22.png)
+
+* After completing the mappings, click *Next* to obtain the below review screen indicating the audience and destination are connected.
+
+![](../../assets/catalog/email-marketing/sendgrid/23.jpg)
+
+* Click *Finish* to complete the setup.
+
+![](../../assets/catalog/email-marketing/sendgrid/24.png)
```

Comment:
I think that the information between screenshots 24-32 is really important, thank you for adding it. 

However, the information refers to **how to validate that the destination has been correctly set up**. Can you please create a new section for this information, so it is easy for readers to scan the page and see what this information is about?

Reply:
Added a Validate section